package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "setup-opentofu",
+  "version": "0.0.1",
+  "description": "Setup OpenTofu CLI for GitHub Actions",
+  "license": "MPL-2.0",
+  "publisher": "OpenTofu",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/opentofu/setup-opentofu.git"
+  },
+  "scripts": {
+    "test": "semistandard --env jest && jest --coverage",
+    "lint": "semistandard --env jest --fix",
+    "build": "ncc build wrapper/tofu.js --out wrapper/dist && ncc build index.js --out dist",
+    "prepare": "husky install",
+    "format-check": "echo \"unimplemented for actions/reusable-workflows basic-validation\""
+  },
+  "keywords": [],
+  "author": "",
+  "dependencies": {
+    "@actions/core": "^1.10.1",
+    "@actions/exec": "^1.1.1",
+    "@actions/io": "^1.1.3",
+    "@actions/tool-cache": "^2.0.1"
+  },
+  "devDependencies": {
+    "@vercel/ncc": "^0.38.0",
+    "husky": "^8.0.3",
+    "jest": "^29.7.0",
+    "nock": "^13.3.3",
+    "semistandard": "^17.0.0"
```

Comment:
IMO Better to pin dependencies, also configured dependabot so it will take care of those

Reply:
I'm not sure I understand the point here. Could you please paraphrase?

Reply:
having `^` means you aren't pinning for a specific version, you ask for this version and above, which can also have some issues if they introduce new bugs

Reply:
Indeed :) The versions config was inherited from setup-terraform. I'll implement the changes you suggested.

Reply:
done